### PR TITLE
draft: Credijusto auth0 blog post

### PR DESCRIPTION
**Pull request by spara**

### CODE REVIEW

File: content/blog/pulumi-auth0/index.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
+---
+title: "Credijusto Manages Authentication with Auth0 and Pulumi"
+date: 2020-09-22
+draft: true
```
Comment (zchase) [inline content/blog/pulumi-auth0/index.md:4]: Does this need to be removed?

File: content/blog/pulumi-auth0/index.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+Create a new Pulumi program:
+
+```bash
+$ pulumi new typescript
+```
+
+Install the Auth0 provider SDK:
+
+```bash
+$ npm add @pulumi/mysql
```
Comment (zchase) [inline content/blog/pulumi-auth0/index.md:33]:
```suggestion
$ npm add @pulumi/auth0
```

File: content/blog/pulumi-auth0/index.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+```
+
+Create the Auth0 resources in index.ts as shown in the example below.
```
Comment (zchase) [inline content/blog/pulumi-auth0/index.md:36]: Should there be an example below this?

**Review by zchase — CHANGES_REQUESTED** [review]
I think there is an example missing and a few updates that need to be made before we can merge this.

File: content/blog/pulumi-auth0/index.md (hunk 1 above)
Comment (spara) [reply content/blog/pulumi-auth0/index.md:4]: yes

File: content/blog/pulumi-auth0/index.md [hunk 4]
```diff
@@ -1,5 +1,5 @@
 ---
-title: "Managing Authentication at Credijusto with the Pulumi Auth0 Provider"
+title: "Credijusto Manages Authentication with Auth0 and Pulumi"
 date: 2020-09-22
 draft: true
```
Comment (spara) [inline content/blog/pulumi-auth0/index.md:4]:
```suggestion
draft: false
```

File: content/blog/pulumi-auth0/index.md [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+$ pulumi new typescript
+```
+
+Install the Auth0 provider SDK:
+
+```bash
+$ npm add @pulumi/auth0
+```
+
+Create the Auth0 resources in index.ts as shown in the example below.
```
Comment (spara) [inline content/blog/pulumi-auth0/index.md:36]:
```suggestion
Create the Auth0 resources in `index.ts` you created as shown in the example below.
```

File: content/blog/pulumi-auth0/index.md (hunk 3 above)
Comment (spara) [reply content/blog/pulumi-auth0/index.md:36]: He's referring to the index.ts that was just created. I changed it a little to clarify it.

**Review by zchase — APPROVED** [review]

**Pull request merged**